src/main/app.ts

```diff
@@ -1083,6 +1085,11 @@ export class KandoApp {
       click: () => this.showEditor(),
     });
 
+    template.push({
+      label: 'Enable/Disable shortcuts',
+      click: () => this.turnShortcuts(),
+    });
+
```

Comment:
I think it would be cooler if the label of the entry changed when clicking it. So initially it would be "Disable All Shortcuts" and once the shortcuts are disabled, it becomes "Enable All Shortcuts". I think this should be possible if you call the `updateTrayMenu()` inside the handler and add a different menu item based on the current shortcut state.